dm/ctl/common/util.go

```diff
@@ -36,10 +38,32 @@ import (
 )
 
 var (
-	masterClient pb.MasterClient
 	globalConfig = &Config{}
+	ctlClient    = &CtlClient{}
 )
 
+// CtlClient used to get master client for dmctl
+type CtlClient struct {
+	tls        *toolutils.TLS
+	etcdClient *clientv3.Client
+}
+
+func (c *CtlClient) getMasterClient() (pb.MasterClient, error) {
+	var (
+		err  error
+		conn *grpc.ClientConn
+	)
+	endpoints := c.etcdClient.Endpoints()
+	for _, endpoint := range endpoints {
+		//nolint:staticcheck
+		conn, err = grpc.Dial(utils.UnwrapScheme(endpoint), c.tls.ToGRPCDialOption(), grpc.WithBackoffMaxDelay(3*time.Second), grpc.WithBlock(), grpc.WithTimeout(3*time.Second))
+		if err == nil {
+			return pb.NewMasterClient(conn), nil
```

Comment:
Do we need to close any previous conn or client instance? or Can we reuse the previous one if it's still valid?